docs/breaking-changes.md

```diff
@@ -14,6 +14,24 @@ This document uses the following convention to categorize breaking changes:
 
 ## Planned Breaking API Changes (22.0)
 
+### Deprecated: `BrowserWindow` APIs
+
+The following APIs have been deprecated:
+
+* `browserWindow.loadURL`
+* `browserWindow.loadFile`
+* `browserWindow.reload`
+* `browserWindow.showDefinitionForSelection`
+* `browserWindow.capturePage`
```

Comment:
Actually having thought about it, I'm ok with these three being deprecated and removed. loadURL/loadFile should stay though.